modules/visual_script/doc_classes/VisualScriptYield.xml

```diff
@@ -1,25 +1,32 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualScriptYield" inherits="VisualScriptNode" version="4.0">
 	<brief_description>
+		A Visual Script node used to pause a function execution.
 	</brief_description>
 	<description>
+		[VisualScriptYield] will pause the function call and return [VisualScriptFunctionState], which can be used to resume the function.
 	</description>
 	<tutorials>
 	</tutorials>
 	<methods>
 	</methods>
 	<members>
 		<member name="mode" type="int" setter="set_yield_mode" getter="get_yield_mode" enum="VisualScriptYield.YieldMode" default="1">
+			The mode to use for yielding. See [enum YieldMode] for available options.
 		</member>
 		<member name="wait_time" type="float" setter="set_wait_time" getter="get_wait_time">
+			The time to wait when [member mode] is set to [constant YIELD_WAIT].
```

Comment:
The instance of a VisualScriptYield does not show this option.
It is used to identify class wait time etc..